lib/test/repo.ex

```diff
@@ -0,0 +1,3 @@
+defmodule EctoTest.Repo do
```

Comment:
Instead, let's move this to `test/fixtures/repo.ex` and add `test/fixtures` to `elixirc_paths` in `mix.exs.

Reply:
To be clear, we need to conditionally add `test/fixtures` to `elixirc_paths`, only when `MIX_ENV=test`